webplugin/js/app/km-widget-custom-events.js

```diff
@@ -80,152 +67,151 @@ var kmWidgetEvents = {
     }
     data property 
     similar format will follow in ga4 as follow
-    {
-        eventCategory: became custom parameter
-        eventAction: became event name
-        eventLabel: became custom parameter
-    }
+    https://developers.google.com/tag-platform/gtagjs/reference#event
+   
 */
 var eventMapping = {
     onChatWidgetOpen: {
         data: {
             eventCategory: 'Kommunicate_widget',
             eventAction: 'km_widget_open',
-            eventLabel: 'Chat Widget Open',
+            eventLabel: 'Km Widget Chat Widget Open',
         },
         eventFunction: null,
     },
     onChatWidgetClose: {
         data: {
             eventCategory: 'Kommunicate_widget',
             eventAction: 'km_widget_close',
-            eventLabel: 'Chat Widget Close',
+            eventLabel: 'Km Widget Chat Widget Close',
         },
         eventFunction: null,
     },
     onRichMessageButtonClick: {
         data: {
             eventCategory: 'Kommunicate_widget',
             eventAction: 'km_widget_rich_message_button_click',
+            eventLabel: 'Km Widget RichMessage Button Click',
         },
         eventFunction: null,
     },
     onFaqClick: {
         data: {
             eventCategory: 'Kommunicate_widget',
             eventAction: 'km_widget_faq_click',
-            eventLabel: 'FAQ Menu',
+            eventLabel: 'Km Widget FAQ Menu',
         },
         eventFunction: null,
     },
     onRateConversationClick: {
         data: {
             eventCategory: 'Kommunicate_widget',
             eventAction: 'km_widget_CSAT_started',
-            eventLabel: 'CSAT Start',
+            eventLabel: 'Km Widget CSAT Start',
         },
         eventFunction: null,
     },
     onSubmitRatingClick: {
         data: {
             eventCategory: 'Kommunicate_widget',
             eventAction: 'km_widget_CSAT_submit',
-            eventLabel: 'CSAT Submit',
+            eventLabel: 'Km Widget CSAT Submit',
         },
         eventFunction: null,
     },
     onShowResolvedClick: {
         data: {
             eventCategory: 'Kommunicate_widget',
             eventAction: 'km_widget_resolve_click',
-            eventLabel: 'Show Resolve',
+            eventLabel: 'Km Widget Show Resolve',
```

Comment:
from where these names came? what was the issue with earlier label?

Reply:
in ga4 only event action is the only field required so i am sending eventCategory, eventLabel as an additional property under that event action so i added this but it is not req to change the eventLabel so i removed the added prefix